src/shared/bg_pmove.cpp

```diff
@@ -2963,15 +2963,7 @@ static void PM_GroundTrace()
 			//toggle wall climbing if holding crouch
 			if ( pm->cmd.upmove < 0 && !( pm->ps->pm_flags & PMF_CROUCH_HELD ) )
 			{
-				if ( pm->ps->stats[ STAT_STATE ] & SS_WALLCLIMBING )
-				{
-					pm->ps->stats[ STAT_STATE ] &= ~SS_WALLCLIMBING;
-				}
-				else
-				{
-					pm->ps->stats[ STAT_STATE ] |= SS_WALLCLIMBING;
-				}
-
+				pm->ps->stats[ STAT_STATE ] ^= ~SS_WALLCLIMBING;
```

Comment:
This toggles every flag except wallclimbing

Reply:
The joy of cut & paste carelessly... sorry, and fixed.